catalogs/catalog-jdbc-postgresql/src/main/java/com/datastrato/gravitino/catalog/postgresql/operation/PostgreSqlTableOperations.java

```diff
@@ -84,139 +55,30 @@ public void initialize(
   @Override
   public JdbcTable load(String schema, String tableName) throws NoSuchTableException {
```

Comment:
could you put the common logic in JdbcTableOperations, leave the specified logic in XXTableOperations?  since Mysql obtain most field from Jdbc drivers too.

Reply:
how about put `createCommonTableBuilder ` in  JdbcCatalogOperations?